jobs/jobs-operator/jobs-operator/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3.11.0
```

Comment:
you might want to consider using a staged build like the other docker images for this, it can make building the image faster/smaller but it's not a big deal